From phil@ipom.com Fri Sep 30 00:28:23 2005
Message-ID: <433CE8E8.2080300@ipom.com>
Date: Fri, 30 Sep 2005 00:27:36 -0700
From: Phil Dibowitz <phil@ipom.com>
To: Greg KH <greg@kroah.com>, Tyson Vinson <lornoss@gmail.com>
Subject: [PATCH pd34] Add unusual_devs for iPod Nano

This adds US_FL_FIX_CAPACITY for yet _another_ entire block of Apple
productIds. They really can't seem to get this right. This one is for
the iPod Nano. Reported by Tyson Vinson <lornoss@gmail.com>.

Signed-off-by: Phil Dibowitz <phil@ipom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

 drivers/usb/storage/unusual_devs.h |   10 ++++++++++
 1 file changed, 10 insertions(+)

--- gregkh-2.6.orig/drivers/usb/storage/unusual_devs.h	2005-10-03 15:04:50.000000000 -0700
+++ gregkh-2.6/drivers/usb/storage/unusual_devs.h	2005-10-03 15:04:57.000000000 -0700
@@ -605,6 +605,16 @@
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
 		US_FL_FIX_CAPACITY ),
 
+/*
+ * Reported by Tyson Vinson <lornoss@gmail.com>
+ * This particular productId is the iPod Nano
+ */
+UNUSUAL_DEV( 0x05ac, 0x120a, 0x0000, 0x9999,
+		"Apple",
+		"iPod",
+		US_SC_DEVICE, US_PR_DEVICE, NULL,
+		US_FL_FIX_CAPACITY ),
+
 #ifdef CONFIG_USB_STORAGE_JUMPSHOT
 UNUSUAL_DEV(  0x05dc, 0x0001, 0x0000, 0x0001,
 		"Lexar",
